e2e/helpers/localStorage.ts

```diff
@@ -0,0 +1,29 @@
+import { Page } from '@playwright/test'
+
+async function navigateToHomepageIfPageHasNotBeenLoaded(page: Page) {
+  if (page.url() === 'about:blank') {
+    // Navigate to homepage, so that localStorage is available
+    await page.goto('/')
+  }
+}
+
+export async function setLocalStorage(page: Page, key: string, value: string) {
+  await navigateToHomepageIfPageHasNotBeenLoaded(page)
+  await page.evaluate(
+    ([key, value]) => {
+      localStorage.setItem(key, value)
+    },
+    [key, value],
+  )
+}
```

Comment:
Navigate to the home page if the page hasn't been loaded, since localStorage isn't available until the page has initially loaded.